opencog/persist/zmq/atomspace/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ ADD_CUSTOM_COMMAND(
 )
 
 ADD_LIBRARY (zmqatoms SHARED
-	ZMQMessages.pb
+	ZMQMessages.pb.cc ZMQMessages.pb.h
```

Comment:
Notice that atomspace builds fine. I get undefined reference errors when trying to link opencog cogserver to libzmqatoms.so. Also, you can check if ZMQMessages.pb.cc is actually built and linked to libzmqatoms.so. 

I use CMake 3.3.2